creator-node/src/config.js

```diff
@@ -665,8 +667,8 @@ const config = convict({
 const defaultConfigExists = fs.existsSync('default-config.json')
 if (defaultConfigExists) config.loadFile('default-config.json')
 
```

Comment:
```suggestion
const pathTo = fileName => path.join(process.cwd(), filename)
```

Reply:
noticed you use this a bit throughout for path resolution; perhaps this can become a util?

Reply:
moved into a func